src/app/feed/feed-header/feed-header.component.scss

```diff
@@ -174,4 +180,8 @@
 		left: 20px !important;
```

Comment:
Could you remove `!important;`, if it is not needed. It seems to be bad practice. And the first answer in this link is too comical 😂  
https://stackoverflow.com/questions/9245353/what-does-important-in-css-mean